core/orchestrator.go

```diff
@@ -581,6 +582,11 @@ func (n *LivepeerNode) transcodeSeg(config transcodeConfig, seg *stream.HLSSegme
 				string(md.ManifestID), md.AuthToken.SessionId, seg.SeqNo, len(tSegments[i].Data))
 			return terr(fmt.Errorf("ZeroSegments"))
 		}
+		if md.CalcPerceptualHash && tSegments[i].PHash == nil {
+			glog.Errorf("Could not find perceptual hash for manifestID=%s sessionID=%s seqNo=%d profile=%v",
+				string(md.ManifestID), md.AuthToken.SessionId, seg.SeqNo, md.Profiles[i].Name)
+			return terr(fmt.Errorf("MissingPerceptualHash"))
```

Comment:
Temporarily remove this, as this might break O-T split case during grace period. Will add this after 14 days grace period is over.